### DISCUSSION TITLE
Update alert query when the query in a panel is changed

### DISCUSSION THREAD

**Discussion by ThommyH** [grafana/grafana#72610] [Alerting]
When you create an alert from an panel the query is suggested for the new alert. However, when you update a panel you must recreate/update the alert to follow suit or else the panel and the alert are out of sync and you may not notice that the alert is not really anymore what you expect.

Do I overlooked something or is it really that cumbersome? Or is there a different, better way?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82492. Feel free to continue the discussion around this there. Thank you!
